instrumentation/lettuce-5.0/src/main/java/io/lettuce/core/AbstractRedisAsyncCommands.java

```diff
@@ -0,0 +1,58 @@
+package io.lettuce.core;
+
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Segment;
+import com.newrelic.api.agent.Trace;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import com.nr.fit.lettuce.instrumentation.NRBiConsumer;
+
+import io.lettuce.core.api.StatefulConnection;
+import io.lettuce.core.protocol.AsyncCommand;
+import io.lettuce.core.protocol.ProtocolKeyword;
+import io.lettuce.core.protocol.RedisCommand;
+
+@Weave
+public abstract class AbstractRedisAsyncCommands<K, V> {
```

Comment:
Same comment about naming weaved library classed with `_Instrumentation` and specifying the `originalName` in the `Weave` annotation.